tapir/coop/views/shareowner.py

```diff
@@ -690,7 +689,7 @@ def shift_slot_filter(queryset: ShareOwner.ShareOwnerQuerySet, name, value: str)
     @staticmethod
     def display_name_filter(queryset: ShareOwner.ShareOwnerQuerySet, name, value: str):
         # This is an ugly hack to enable searching by Mitgliedsnummer from the
-        # one-stop search box in the top right
+        # one-stop search box in the t  op right
```

Comment:
That looks like a typo :p
```suggestion
        # one-stop search box in the top right
```

Reply:
sorry, fixed